File: leetcode/319_bulb_switcher.cpp
// Copyright (c) 2018 Jacopo Notarstefano

#include <cassert>
#include <cmath>

using namespace std;

class Solution {
 public:
    int bulbSwitch(int n) {
        return sqrt(n);
    }
};

int main() {
    auto solution = Solution();

    assert(1 == solution.bulbSwitch(3));
}

File: leetcode/319_bulb_switcher.py
# -*- coding: utf-8 -*-

from __future__ import absolute_import, division, print_function

import math


class Solution(object):
    def bulbSwitch(self, n):
        return int(math.sqrt(n))


if __name__ == '__main__':
    solution = Solution()

    assert 1 == solution.bulbSwitch(3)
